MAUI/Popup/layout-customizations.md

```diff
@@ -317,6 +317,9 @@ public partial class MainPage : ContentPage
 
 ### Popup header height
 
+N>
+When the HeaderTemplate is set, the HeaderHeight property is used to customize the header height.

```

Comment:
When the HeaderTemplate is set, the HeaderHeight property customizes the header height.

Reply:
I have Addressed review comments.